### DISCUSSION TITLE
how to display job progress

### DISCUSSION THREAD

**Discussion by alvaromartmart** [community/community#160739] [Actions]
Hi, I was wondering how can I display job execution progress in the workflow run preview. The figure below, [found here](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows), shows what I'm referring to. Is this an experimental feature? Couldn't find anywhere in the docs how to achieve this.

![image](https://user-images.githubusercontent.com/18287793/143570032-ca006e1c-0272-4c32-9408-457b461e0872.png)

**Comment by sbrakl** [comment]
https://stackoverflow.com/questions/72912540/how-can-i-add-progress-bar-to-my-github-action
